README.md

```diff
@@ -1,64 +1 @@
-# Links
-- Slack group: https://app.slack.com/client/T024FPYBQ/C03BUFQB50V/T2VHPBHA5
-- Figma: https://www.figma.com/file/Q7LqTYZxPqUVjD8tF7ihmy/FSA-Capstone
-- Todo Board: https://github.com/orgs/Team5-Capstone/projects/1/views/1
-
```

Comment:
This url doesn't go anywhere, and `orgs/testbrew` is `testbrew/testbrew`
https://github.com/orgs/testbrew/testbrew 

try changing it to

[TestBrew Repo](https://github.com/testbrew/testbrew)
with a link

Reply:
Nice, last commit should fix that.